cdk/dcm_floodlight/transform.js

```diff
@@ -19,14 +18,15 @@ function appendProperties(endpoint, payload) {
 
 // transform webapp dynamicForm custom floodlight variable
 // into {u1: value, u2: value, ...}
-function transformCustomVariable(customFloodlightVariable) {
+function transformCustomVariable(customFloodlightVariable, message) {
   const customVariable = {};
   customFloodlightVariable.forEach(item => {
     if (item && !isEmpty(item.from) && !isEmpty(item.to)) {
       // remove u if already there
-      customVariable[
-        `u${item.from.trim().replace(/u/g, "")}`
-      ] = `[${item.to.trim()}]`;
+      const itemValue = get(message, `properties.${item.to.trim()}`);
+      if (itemValue) {
+        customVariable[`u${item.from.trim().replace(/u/g, "")}`] = itemValue;
+      }
     }
```

Comment:
We should handle for 0, false , "", null values. The competition is removing in case presence of such values.
Also we should once verify handling of boolean values liek false and true is boolean supported in dcm?

Reply:
Handled all the cases. Supported types are number and string

Reply:
We are omitting boolean values doesn't dcm support boolean values?